src/components/CategoryPage.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import { Box, Grid, GridItem } from "@chakra-ui/react";
+
+import { filters, products } from "~/data/data";
+import useFilter from "~/hooks/useFilter";
+import type { Product, ProductType } from "~/types/product";
+import FilterGroup from "./FilterGroup";
+import ProductList from "./ProductList";
+import { SortGroup } from "./SortGroup";
+
+const CategoryPage = ({ product }: { product: ProductType }) => {
+  const [productList, setProductList] = useState<Product[]>(products[product]);
+  const [setFilter, setSortOption] = useFilter(products[product], setProductList);
+
+  useEffect(() => {
+    setProductList(products[product]);
+  }, [product]);
```

Comment:
when will the dependency `product` be changed?

Reply:
this is to make the category tabs work, so now it's changing what `product` is but staying on the same page

Reply:
i think using getStaticProps and getStaticPaths in CategoryPage and pass the product prop down is better. This useEffect makes the About page act like a CategoryPage with filter on the side.